.github/workflows/testing.yml

```diff
@@ -0,0 +1,49 @@
+name: Testing
+# in this file we describe the action to execute for the CI in the dev pipeline
+
+# based on default laravel workflow
+# https://github.com/actions/starter-workflows/blob/master/ci/laravel.yml
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - master
```

Comment:
`master` isn't a testing branch

Reply:
Fixed in b676ff19938de25c22434cd143c96c4842ae2dbd